hbase/README.md

```diff
@@ -47,6 +47,20 @@ Please see the general instructions in the `doc` folder if you are not sure how
 bin/ycsb run hbase -P workloads/workloada -cp /HBASE-HOME-DIR/conf -p table=usertable -p columnfamily=family -p clientbuffering=true
 ```
 
+### 5. Kerberos
```

Comment:
nit: It might be clearer that this is only needed sometimes if the head is "Kerberos (optional)"